Cargo.lock

```diff
@@ -490,7 +490,7 @@ dependencies = [
  "prometheus 0.4.2 (registry+https://github.com/rust-lang/crates.io-index)",
  "protobuf 2.0.4 (registry+https://github.com/rust-lang/crates.io-index)",
  "quick-error 1.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
- "raft 0.4.0 (registry+https://github.com/rust-lang/crates.io-index)",
+ "raft 0.4.2 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
seem kvproto can use 0.4.2 directly? no need to bind a fixed git, right? @nrc

Reply:
I guess that's because kvproto in TiKV is not updated yet.